src/app/client/src/app/modules/core/components/prominent-filter/prominent-filter.component.ts

```diff
@@ -228,6 +229,13 @@ export class ProminentFilterComponent implements OnInit, OnDestroy {
   isObject(val) { return typeof val === 'object'; }
 
   applyFilters() {
+    const compareFormInputQueryparams = _.isEqual(this.formInputData,  this.queryParams);
```

Comment:
do we need the constant as it is only used in a if

Reply:
removed assigning to a constant